index.css

```diff
@@ -300,8 +300,13 @@ button{
     border-radius: 5px;
     padding: 12px;
     width: 100%;
+    outline: none;
 
 }
+.form__item input::placeholder{
```

Comment:
here as well , avoit usage of html tag in css script . kindly use class name instead